### DISCUSSION TITLE
Typekit / Adobe Font support

### DISCUSSION THREAD

**Discussion by tablufc88** [vercel/next.js#52877] [App Router]
Is there any way we can use Adobe Fonts/Typekit with the app dir?

As per [the docs](https://nextjs.org/docs/app/building-your-application/styling/css-modules#external-stylesheets), `<link rel="stylesheet" />` is unusable. We have a client who needs one of these fonts & although we could pay to licence it and then load it in using next/font/local, we would rather just use our Typekit subscription.

**Comment by peperoli** [comment]
I'm also interested in this, we have several existing projects using Adobe Fonts we're considering to migrate to the App Router.

**Comment by maxscn** [comment]
Would like this as well. Currently using the old head tags as below. 

```
{/* layout.tsx */}
<head>
  <link rel="stylesheet" href="https://use.typekit.net/path.css" />
</head>
{/* ... */}
```

**Comment by Ojay** [comment]
Also interested in this, any news at all?? Thanks!

**Reply by huyb1991** [reply]
It support on Canary version, you can take a look on this PR: https://github.com/vercel/next.js/pull/24834/files

**Comment by Jabsolu23** [comment]
Interested in this also
